src/views/AgmLocationChg.vue

```diff
@@ -634,7 +632,8 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     const data: any = {
       [FilingTypes.AGM_LOCATION_CHANGE]: {
         year: this.agmYear,
-        newAgmLocation: this.agmLocationAddress
+        reason: this.reason,
+        agmLocation: this.agmLocation
```

Comment:
Was looking at Hongjing's PR: https://github.com/bcgov/business-schemas/pull/146

Not sure if it'll change later but for now, looks like this is what we need.